velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2858,6 +2858,41 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
 
+TEST_F(DateTimeFunctionsTest, currentTimestamp) {
```

Comment:
You can combine currentTimestamp and now tests by modifying this lamda to also take function name as a parameter and invoke it with both currentTimestamp and now in the same test. That would make the code more compact and readable.

Reply:
Do we have an example in tests/DateTimeFunctionsTest.cpp? I didn't see people doing that.

Reply:
There might not be... I was suggesting this

  const auto evaluateNow = [&](const std::string& functionName, 
                              int64_t expectedTimestamp,
                               const std::string& timeZone) {
    VectorPtr expected =
        makeTimestampWithTimeZoneVector(expectedTimestamp, timeZone.c_str());
    VectorPtr actual = evaluateWithTimestampWithTimezone( functionName, 0, "+00:00");

    // Accept 5 milliseconds lag since sometimes the machine is slow.
    ASSERT_TRUE(actual->compare(expected.get(), 0, 0) < 5)
    ": expected "
        << expected->toString(0) << ", but got " << actual->toString(0)
        << ", which is larger than the 5 milliseconds acceptable lag.";
  };

evaluateCurrentTimestamp("currentTimestamp()", Timestamp::now().toMillis(), "+00:00");
evaluateCurrentTimestamp("now()", Timestamp::now().toMillis(), "+00:00");

Reply:
Ok, that's a good idea, I will have a try.